remix-astwalker/tests/resources/ast.ts

```diff
@@ -0,0 +1,27 @@
+import { Node } from '../../src/'
+let node: Node = {
```

Comment:
Here you are never reassigning `node` (only mutating it) use `const` instead of `let`.